lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -735,49 +729,47 @@
       [{:loc (z/replace source switch)
         :range (meta (z/node source))}])))
 
-(defn inline-symbol?
-  [{:keys [filename name-row name-col bucket]} db]
-  (when (or (identical? :locals bucket)
-            (identical? :var-definitions bucket))
-    (some-> (parser/safe-zloc-of-file db (shared/filename->uri filename db))
-            (parser/to-pos name-row name-col)
-            edit/find-op
-            z/sexpr
-            #{'let 'def})))
+(defn ^:private inline-data [uri row col db]
+  (let [{:keys [uri bucket name-row name-col] :as definition} (q/find-definition-from-cursor db uri row col)]
+    (when (or (identical? :locals bucket)
+              (identical? :var-definitions bucket))
+      (when-let [zloc (some-> (parser/safe-zloc-of-file db uri)
+                              (parser/to-pos name-row name-col))]
+        (when-let [op (some-> zloc edit/find-op z/sexpr #{'let 'def})]
+          {:def-elem definition
+           :def-zloc zloc
+           :def-uri uri
+           :def-op op})))))
```

Comment:
Again, remove some duplication between `inline-symbol?` (and code-actions) and `inline-symbol`.